README.md

```diff
@@ -1,4 +1,4 @@
-<!-- README.md is generated from README.Rmd. Please edit that file -->
+<!-- README.md is generated from README.Rmd. Please edit README.Rmd -->
```

Comment:
@MonkmanMH Wondering if it is possible to knit the .Rmd file so that .md file matches and then it is all tied to your commits and this PR?